File: rust/2024/src/main.rs
use aoc2024::days;

// TODO: Add library to be able to run doctests.
fn main() {
    aoc2024::Solver::from("../inputs/2024")
        .add::<days::day01::Solution>()
        .add::<days::day02::Solution>()
        .add::<days::day03::Solution>()
        .add::<days::day04::Solution>()
        .compute()
}

File: zig/src/2024/main.zig
const std = @import("std");
const Runner = @import("advent_utils.zig").Runner;
const days = @import("days/index.zig");

pub fn main() !void {
    try Runner("../inputs/2024/").add(days.day01.Solution).add(days.day02.Solution).run();
}
